<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>660</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>SexySolver Astrometry Tester</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="LoadImage">
        <property name="text">
         <string>Load Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomIn">
        <property name="text">
         <string>Zoom In</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomOut">
        <property name="text">
         <string>Zoom Out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="AutoScale">
        <property name="text">
         <string>Auto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showStars">
        <property name="text">
         <string>ShowStars</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QToolBox" name="optionsBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="Basic">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>197</width>
          <height>371</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Basic</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="4" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Images Path</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Solver Path</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="solverPath"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="sextractorPath"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="imagesDefaultPath"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Sextractor Path</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Temp Path</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="tempPath"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Sextractor">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>197</width>
          <height>371</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Sextractor</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="2">
          <widget class="QScrollArea" name="scrollArea_4">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_4">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>185</width>
              <height>345</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_9">
             <item row="12" column="1">
              <widget class="QComboBox" name="star_shape">
               <item>
                <property name="text">
                 <string>Auto</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Circle</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Ellipse</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="r_min_3"/>
             </item>
             <item row="16" column="1">
              <widget class="QLineEdit" name="r_min"/>
             </item>
             <item row="16" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>r_min</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Deblend</string>
               </property>
              </widget>
             </item>
             <item row="17" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>magzero</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Min Area</string>
               </property>
              </widget>
             </item>
             <item row="17" column="1">
              <widget class="QLineEdit" name="r_min_2"/>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_14">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Photometry</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Min Contrast</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>N_Thresh</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="r_min_4"/>
             </item>
             <item row="12" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Shape</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Threshhold</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="r_min_7"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_16">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Extraction</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLineEdit" name="r_min_6"/>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="checkBox">
               <property name="text">
                <string>Clean?</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="r_min_5"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Astrometry">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>197</width>
          <height>371</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Astrometry</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="2" column="0">
          <widget class="QScrollArea" name="scrollArea_3">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_3">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>195</width>
              <height>369</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="leftMargin">
              <number>4</number>
             </property>
             <property name="topMargin">
              <number>4</number>
             </property>
             <property name="rightMargin">
              <number>4</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>-1</number>
             </property>
             <item row="0" column="2">
              <widget class="QLineEdit" name="configFilePath"/>
             </item>
             <item row="11" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>RA</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QCheckBox" name="use_scale">
               <property name="text">
                <string>Use Scale</string>
               </property>
              </widget>
             </item>
             <item row="13" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Radius</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="scale_low"/>
             </item>
             <item row="10" column="2">
              <widget class="QCheckBox" name="use_position">
               <property name="text">
                <string>Use Position</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Low</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Units</string>
               </property>
              </widget>
             </item>
             <item row="11" column="2">
              <widget class="QLineEdit" name="ra"/>
             </item>
             <item row="13" column="2">
              <widget class="QLineEdit" name="radius"/>
             </item>
             <item row="8" column="2">
              <widget class="QComboBox" name="units">
               <item>
                <property name="text">
                 <string>app</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>aw</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Config File</string>
               </property>
              </widget>
             </item>
             <item row="12" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>DEC</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QLineEdit" name="scale_high"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>High</string>
               </property>
              </widget>
             </item>
             <item row="12" column="2">
              <widget class="QLineEdit" name="dec"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="fileNameDisplay">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>197</width>
              <height>114</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="ImageLabel" name="Image">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>199</width>
           <height>330</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item row="0" column="0">
           <widget class="QTabWidget" name="tabWidget">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="logOutput">
             <attribute name="title">
              <string>Log Output</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_6">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QTextBrowser" name="logDisplay"/>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="solutionTableView">
             <attribute name="title">
              <string>Solution Table</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_7">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QTableWidget" name="solutionTable"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QTableWidget" name="starList"/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="SextractStars">
        <property name="text">
         <string>Sextract</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="InnerSextract">
        <property name="text">
         <string>Sextract Internal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SolveImage">
        <property name="text">
         <string>Solve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="InnerSolve">
        <property name="text">
         <string>Solve Internal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Abort">
        <property name="text">
         <string>Abort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ClearLog">
        <property name="text">
         <string>ClearLog</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSextractor_and_Astrometry_Tester">
    <property name="title">
     <string>SexySolver Astrometry Tester</string>
    </property>
   </widget>
   <addaction name="menuSextractor_and_Astrometry_Tester"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>imagelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
